### PR TITLE
Target _blanks + miscellaneous

### PR DESCRIPTION
**Pull request by JulienZD**
* Remove Twitter
* Improve window title
* Remove manual markdown parsing

### CODE REVIEW

File: src/components/Projects.tsx [hunk 1]
```diff
@@ -16,7 +16,9 @@ export default function Projects({ projects, projectsRef }: Props): JSX.Element
       <p>
         Note: This list isn't up-to-date. Check my{' '}
         <Link href="https://github.com/JulienZD/">
-          <a className="link-animated-hover">Github</a>
+          <a className="link-animated-hover" target="_blank">
+            Github
```
Comment (RobinHeidenis) [inline src/components/Projects.tsx:20]: moet hier niet nog het <Github /> element in?

Comment (RobinHeidenis) [reply src/components/Projects.tsx:20]: bruh removed <Github />

Comment (RobinHeidenis) [reply src/components/Projects.tsx:20]: W HAT

Comment (RobinHeidenis) [reply src/components/Projects.tsx:20]: < Github />

Comment (RobinHeidenis) [reply src/components/Projects.tsx:20]: thanks

Comment (JulienZD) [reply src/components/Projects.tsx:20]: No, this is just a link mentioning Github as text 
image

**Review by RobinHeidenis — APPROVED** [review]

**Pull request merged**